Guided Hunting - Office365-Exploring.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>Reference before assignment - think this might be left over from debugging</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Removed</p>